app/controllers/meetings/edit/treatment.js

```diff
@@ -0,0 +1,136 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+import { inject as service } from '@ember/service';
+
+const DRAFT_DECISION_FOLDER_ID = 'ae5feaed-7b70-4533-9417-10fbbc480a4c';
+const GEAGENDEERD_STATUS_ID = '7186547b61414095aa2a4affefdcca67';
+
+export default class MeetingsEditTreatmentController extends Controller {
+  @service router;
+  @service currentSession;
+  @tracked editor;
+  @tracked documentContainer;
+  @tracked document;
+  @tracked initialContent;
+  @tracked confirmLeaveOpen;
+
+  get dirty() {
+    return this.document.content !== this.editor.htmlContent;
+  }
+
+  @action
+  closeModal() {
+    this.router.transitionTo('meetings.edit', this.model.meetingId);
+  }
+
+  @action
+  handleRdfaEditorInit(editor) {
+    if (this.document.content) {
+      editor.setHtmlContent(this.document.content);
+    }
+    this.editor = editor;
+  }
+
+  @action
+  async saveAndQuit() {
+    await this.saveDocumentTask.perform();
+    this.closeModal();
+  }
+  @action
+  async cancel() {
+    const container = await this.documentContainer;
+    container.rollbackAttributes();
```

Comment:
I don't think the rollback is needed here since the confirm behavior will handle that?